controllers/topics.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { selectTopics } = require('../models/topics.models');
+
+const getAllTopics = (req, res, next) => {
+  selectTopics().then((data) => {
+    res.status(200).send(data);
+  });
```

Comment:
If you're adding next make sure it's accompanied by a catch block with a next piece of error handling middleware, otherwise just leave as req and res until needed.